### ISSUE TITLE
Reproduceable: Cannot set property defaults of #<o> which has only a getter on latest version of GrapesJS

### ISSUE THREAD

**Issue by mitcht** [GrapesJS/components-lory#18]
https://jsfiddle.net/0srph1k7/

Adding the release version of the GrapesJS Lory Slider Plugin results in an error.  Cannot set property defaults of #<o> which has only a getter 

I've tried passing the default method into the grapesjs plugin array, but the result is still the same.

Tested in Chrome Version 121.0.6167.185 (Official Build) (64-bit)
Tested in Brave [Version 1.62.165 Chromium: 121.0.6167.184 (Official Build) (64-bit)](https://brave.com/latest/)
Tested in Edge Version 121.0.2277.112 (Official build) (64-bit)

See fiddle above.

**Comment by rdhathaliya** [comment]
TypeError: Cannot set property defaults of #<o> which has only a getter

Same issue!, Did you get any solution ?

**Comment by CarComp** [comment]
@rdhathaliya no. He won't address it. You will have to go in and manually clean up any attributes on the object itself. It kind of sucks but it works. Try comparing the json from your use case to what comes out of the grapes demo page. It should be relatively obvious which objects you need to remove.

**Comment by rdhathaliya** [comment]
@CarComp Thanks for your response but, not getting your point :(

**Comment by valerito** [comment]
Make sure you have the latest version of GrapesJS

**Comment by Xliff** [comment]
Where is a working demo of component-lory for GrapeJS? The current demo does not look like it uses a slider, and I'm getting the same error when `grapesjs-lory-slider` is enabled.

**Comment by CarComp** [comment]
> Where is a working demo of component-lory for GrapeJS? The current demo does not look like it uses a slider, and I'm getting the same error when `grapesjs-lory-slider` is enabled.

This Lory Slider module is woefully outdated.  I don't have better news. Good luck.
